4.8/runtime/windowsservercore-ltsc2019/Dockerfile

```diff
@@ -10,6 +10,20 @@ RUN powershell -Command `
     del .\dotnet-framework-installer.exe & `
     powershell Remove-Item -Force -Recurse ${Env:TEMP}\*
 
+# Apply latest patch
+RUN powershell -Command `
+        $ErrorActionPreference = 'Stop'; `
+        $ProgressPreference = 'SilentlyContinue'; `
+        Invoke-WebRequest `
+            -UseBasicParsing `
+            -Uri "http://download.windowsupdate.com/d/msdownload/update/software/secu/2019/05/windows10.0-kb4495590-x64_d746e135ff19912a8fd20a53bcc89fd20cbd3b6a.msu" `
```

Comment:
This doesn't look correct to me.  Shouldn't this be http://download.windowsupdate.com/d/msdownload/update/software/secu/2019/05/windows10.0-kb4495618-x64_2d3245836cfd2467024a907947837e7879463a3b.msu?

Reply:
Yep, that looks better.